tests/functional/comment/__init__.py

```diff
@@ -116,7 +116,7 @@ def teardown_package():
     global TEST_WORKSPACE
 
     print("Removing: " + TEST_WORKSPACE)
-    shutil.rmtree(TEST_WORKSPACE)
+    shutil.rmtree(TEST_WORKSPACE, ignore_errors=True)
```

Comment:
Please remove this change as it is irrelevant to this already big pull request...

... and accept the fact that the comment tests right now can be nondeterministic.

This change has resulted in the whole test suite being botched on macOS. (This test suite is very special as it runs with custom `HOME` and such... removing a folder prematurely results on macOS and SLES in a total inability to run commands, in this case, the server for the `comment` test suite can't stand up, or crashes prematurely.)

I have this exact issue properly figured out in #773, where these failures won't be apparent anymore.